platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/views/LearnerPushMerkleTreeView.java

```diff
@@ -59,15 +59,19 @@ public class LearnerPushMerkleTreeView implements LearnerTreeView<MerkleNode> {
     private final Queue<ExpectedLesson<MerkleNode>> expectedLessons;
     private final LinkedList<MerkleInternal> nodesToInitialize;
 
+    private final ReconnectMapStats mapStats;
+
     /**
      * Create a new standard tree view out of an in-memory merkle tree (or subtree).
      *
      * @param root
      * 		the root of the tree (or subtree)
      */
-    public LearnerPushMerkleTreeView(final ReconnectConfig reconnectConfig, final MerkleNode root) {
+    public LearnerPushMerkleTreeView(
+            final ReconnectConfig reconnectConfig, final MerkleNode root, final ReconnectMapStats mapStats) {
```

Comment:
missing doc and nullity annotation for new param

Reply:
added